apps/channels/models.py

```diff
@@ -11,9 +11,14 @@
 from apps.utils.models import BaseModel
 from apps.web.meta import absolute_url
 
+WEB = "web"
```

Comment:
These names are the same as
```python
PLATFORM = (("telegram", "telegram"), ("web", "web"), ("whatsapp", "whatsapp"))
```

Any particular reason why you're creating these constants then? It would make sense to do this if it were used somewhere else in the code maybe, but personally I don't see reason to do it?

Reply:
I get anxious seeing strings lying around in code, so I like to tie it to constants.

Reply:
I don't have any beef with this